demo/demo.js

```diff
@@ -126,60 +125,138 @@ ethSignMsg.addEventListener("click", async () => {
     }
 });
 
+// Display single-sig BTC address on device for verification
 document.getElementById("btcAddressSimple").addEventListener("click", async () => {
-    await device.bitbox02API.btcDisplayAddressSimple(
-        api.firmware.messages.BTCCoin.BTC,
+    await device.api.btcDisplayAddressSimple(
+        constants.messages.BTCCoin.BTC,
         getKeypathFromString("m/49'/0'/0'/0/0"),
-        api.firmware.messages.BTCScriptConfig_SimpleType.P2WPKH_P2SH,
+        constants.messages.BTCScriptConfig_SimpleType.P2WPKH_P2SH,
     );
 });
 
-document.getElementById("btcSignSimple").addEventListener("click", async () => {
-    const bip44Account = 0 + HARDENED;
-    const version = 1;
-    const locktime = 0;
+// Mock sample single-sig BTC transaction
+function makeExampleTx(keypathAccount) {
```

Comment:
Same as above, I think we should make some demo-utils file and move these `makeTx` functions there